Specs/Scene/ModelExperimental/ModelExperimentalSpec.js

```diff
@@ -1134,1164 +995,2219 @@ describe(
-        scene.renderForSpecs();
-        expect(updateModelMatrix).toHaveBeenCalled();
-        verifyRender(model, false);
+      it("changing sphericalHarmonicCoefficients works", function () {
+        if (!scene.highDynamicRangeSupported) {
+          return;
+        }
+        const L00 = new Cartesian3(
+          0.692622075009195,
+          0.4543516001819,
+          0.36910172299235
+        ); // L00, irradiance, pre-scaled base
+        const L1_1 = new Cartesian3(
+          0.289407068366422,
+          0.16789310162658,
+          0.106174907004792
+        ); // L1-1, irradiance, pre-scaled base
+        const L10 = new Cartesian3(
+          -0.591502034778913,
+          -0.28152432317119,
+          0.124647554708491
+        ); // L10, irradiance, pre-scaled base
+        const L11 = new Cartesian3(
+          0.34945458117126,
+          0.163273486841657,
+          -0.03095643545207
+        ); // L11, irradiance, pre-scaled base
+        const L2_2 = new Cartesian3(
+          0.22171176447426,
+          0.11771991868122,
+          0.031381053430064
+        ); // L2-2, irradiance, pre-scaled base
+        const L2_1 = new Cartesian3(
+          -0.348955284677868,
+          -0.187256994042823,
+          -0.026299717727617
+        ); // L2-1, irradiance, pre-scaled base
+        const L20 = new Cartesian3(
+          0.119982671127227,
+          0.076784552175028,
+          0.055517838847755
+        ); // L20, irradiance, pre-scaled base
+        const L21 = new Cartesian3(
+          -0.545546043202299,
+          -0.279787444030397,
+          -0.086854000285261
+        ); // L21, irradiance, pre-scaled base
+        const L22 = new Cartesian3(
+          0.160417569726332,
+          0.120896423762313,
+          0.121102528320197
+        ); // L22, irradiance, pre-scaled base
+        const coefficients = [L00, L1_1, L10, L11, L2_2, L2_1, L20, L21, L22];
+        return loadAndZoomToModelExperimental(
+          {
+            gltf: boxTexturedGltfUrl,
+            imageBasedLighting: new ImageBasedLighting({
+              sphericalHarmonicCoefficients: coefficients,
+            }),
+          },
+          scene
+        ).then(function (model) {
+          scene.highDynamicRange = true;
 
-        model.scale = 1.0;
-        scene.renderForSpecs();
-        expect(updateModelMatrix).toHaveBeenCalled();
-        verifyRender(model, true);
+          const renderOptions = {
+            scene: scene,
+            time: defaultDate,
+          };
+
+          let result;
+          verifyRender(model, true);
+          expect(renderOptions).toRenderAndCall(function (rgba) {
+            result = rgba;
+          });
+
+          const ibl = model.imageBasedLighting;
+          ibl.sphericalHarmonicCoefficients = undefined;
+          expect(renderOptions).toRenderAndCall(function (rgba) {
+            expect(rgba).not.toEqual(result);
+          });
+
+          scene.highDynamicRange = false;
```

Comment:
This should go in an `afterEach` (or alternatively, set it to the default in `beforeEach`), this way if this test throws an exception it doesn't impact other tests that use the same scene